lib/db/mem.js

```diff
@@ -121,9 +121,11 @@ module.exports = function (log, error) {
 
     if (sessionToken.tokenVerificationId) {
       unverifiedTokens[tokenId] = {
-        tokenVerificationId: sessionToken.tokenVerificationId,
         mustVerify: !! sessionToken.mustVerify,
-        uid: sessionToken.uid
+        tokenVerificationId: sessionToken.tokenVerificationId,
+        tokenVerificationCodeHash: sessionToken.tokenVerificationCode,
```

Comment:
IIUC, this is not actually hashing the `tokenVerificationCode`, so the field name here is wrong.  Should it hash it, for consistency with the mysql version?